packages/components/src/Autocomplete/Autocomplete.types.ts

```diff
@@ -567,6 +567,14 @@ interface AutocompleteRebuiltBaseProps<
    */
   readonly size?: InputTextRebuiltProps["size"];
 
+  /**
+   * Controls the visibility of the clear button on the input.
+   * - "never": never show the clear button
+   * - "while-editing": show while the field has value and is focused
+   * - "always": show when the field has value, even when blurred
+   */
```

Comment:
do you need to redefine the JSDocs if we are leveraging the existing definition?

Reply:
ah I see what you mean

what I was thinking is that if we have
`InputTextRebuiltProps["clearable"];`

can we move this JSDoc into InputText's `clearable` definition so that we write it once and have it applied to both instances? or even one step further, if InputText extends some even lower level shared definition that they all use